api/factories.py

```diff
@@ -0,0 +1,29 @@
+import datetime
+
+import factory.fuzzy
+import pytz
+from .models import *
+
+
+class EventFactory(factory.Factory):
+    class Meta:
+        model = Event
+
+    name = factory.Sequence(lambda n: 'event ' + str(n))
+    description = factory.sequence(lambda n: 'description ' + str(n))
+
+
+class PastEventFactory(EventFactory):
+    datetime = factory.fuzzy.FuzzyDateTime(datetime.datetime(2011, 8, 15, 8, 15, 12, 0, pytz.UTC),
```

Comment:
Why do you use this FuzzyDateTime here? And also, shouldn't it be replace with Faker (as noted in the Factory Boy documentation)?

Reply:
I would also change the date to something like `datetime.today() - datetime.timedelta(days=x)` to keep this hardcoded date off from here.